src/clr-core/tag/tag.element.scss

```diff
@@ -1,34 +1,29 @@
 @import './../styles/mixins/mixins';
-@import './../badge/properties';
-@import './variables';
-@import './properties';
 
 :host {
-  //default
-  $label-nudge-for-vertical-alignment: $clr_baselineRem_1px;
-  @include css-var(font-size, clr-tag-font-size, $clr-label-font-size);
-  @include css-var(font-weight, clr-tag-font-weight, $clr-label-font-weight);
-  @include css-var(letter-spacing, clr-tag-letter-spacing, $clr-label-letter-spacing);
-
-  // line-height needed to be adjusted to force center alignment
-  line-height: $clr-label-font-size + $clr_baselineRem_1px;
+  --font-size: #{$clr-p7-font-size};
+  --font-weight: #{$clr-p7-font-weight};
+  --letter-spacing: #{$clr-p7-letter-spacing};
+  --border-radius: #{$clr_baselineRem_0_5};
+  --background: none;
+  --background-hover: #{$clr-color-neutral-200};
```

Comment:
I think I understand why you want to do this. Have you considered...
`var(--clr-color-neutral-200, #{$clr-color-neutral-200})`

So that way you get the default in there while allowing for an override?

Reply:
Updated to use color vars with sass fallbacks.